CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [2.0.0] - not released yet
 
+### Bug fixes
+ - Now a thread/process is not delete when the task submitted take more time than maxInactiveTime configured( issue #70)
```

Comment:
```suggestion

- Now a thread/process is not delete when the task submitted take more time than `maxInactiveTime` configured (#70)

```

I see a lot of _wrongly_ placed braces in your writing, like this `[...]configured( issue [...]`
Could you try to improve these?

I don't want to blame :sweat_smile:  
I'm just an author of a formatting plugin and so I see such formatting / style mismatches instantly :eyes:  
Formatting and style is not just formatting js code, but the whole code base / all files

Reply:
No problem you are right, I just shared with you and jerome the first working version without review the changes too much, I was interested a lot on show you how we implemented this.

Anyway it's fine :)